app/views/sephora_style_guide/home/_sprite_icons.slim

```diff
@@ -1,169 +1,16 @@
s":"ph","size":"icon_30x20"},{"name":"Malaysia flag","class":"my","size":"icon_30x20"},{"name":"Australia flag","class":"au","size":"icon_30x20"},{"name":"New-Zealand flag","class":"nz","size":"icon_30x20"},{"name":"Thailand flag","class":"th","size":"icon_30x20"},{"name":"Indonesia flag","class":"id","size":"icon_30x20"},{"name":"2C2P 123","class":"twoc2p","size":"icon_50x30"},{"name":"Union Pay","class":"unionpay","size":"icon_50x30"},{"name":"BCA","class":"bca","size":"icon_50x30"},{"name":"MOL Pay","class":"molpay","size":"icon_50x30"},{"name":"AMERICAN EXPRESS","class":"amex","size":"icon_50x30"},{"name":"Mastercard","class":"mastercard","size":"icon_50x30"},{"name":"VISA","class":"visa","size":"icon_50x30"},{"name":"WEB CASH","class":"webcash","size":"icon_50x30"},{"name":"AmBank","class":"ambank","size":"icon_50x30"},{"name":"FPX","class":"fpx","size":"icon_50x30"},{"name":"PayPal","class":"paypal","size":"icon_50x30"},{"name":"TMB","class":"tmb","size":"icon_50x30"},{"name":"KASIKORNBANK","class":"kkbank","size":"icon_50x30"},{"name":"Krungsri","class":"krungsri","size":"icon_50x30"},{"name":"SCB","class":"scb","size":"icon_50x30"},{"name":"Card 16","class":"card16","size":"icon_50x30"},{"name":"UOB","class":"uob","size":"icon_50x30"},{"name":"PAYATPOST","class":"payatpost","size":"icon_50x30"},{"name":"TESCO","class":"tesco","size":"icon_50x30"},{"name":"BBL","class":"bbl","size":"icon_50x30"},{"name":"TrueMoney","class":"truemoney","size":"icon_50x30"},{"name":"T Bank","class":"tbank","size":"icon_50x30"},{"name":"Big C","class":"bigc","size":"icon_50x30"},{"name":"MPay","class":"mpay","size":"icon_50x30"},{"name":"Just Pay","class":"jautpay","size":"icon_50x30"},{"name":"Public Bank","class":"pubbank","size":"icon_50x30"},{"name":"GCash","class":"gcash","size":"icon_50x30"},{"name":"Convenience Stores","class":"cstore","size":"icon_50x30"},{"name":"Mandiri","class":"mandiri","size":"icon_70x30"},{"name":"PayPal","class":"paypal","size":"icon_70x30"},{"name":"VISA","class":"visa","size":"icon_70x30"},{"name":"RHB Now","class":"rhbnow","size":"icon_70x30"},{"name":"MEPF Currency","class":"mepf","size":"icon_70x30"},{"name":"Dragonpay","class":"dragonpay","size":"icon_70x30"},{"name":"Cash On Delivery","class":"cod","size":"icon_70x30"},{"name":"BAYAR DI TEMPAT","class":"bayar","size":"icon_70x30"},{"name":"AfterPay","class":"afterpay","size":"icon_70x30"},{"name":"Hongleong Connect","class":"connect","size":"icon_70x30"},{"name":"PAYATALL","class":"payatall","size":"icon_70x30"},{"name":"maybank2u.com","class":"maybank","size":"icon_100x30"},{"name":"Hong Leong Bank","class":"hongleong","size":"icon_100x30"},{"name":"CIMB Clicks","class":"cimbclick","size":"icon_100x30"},{"name":"Permata Bank","class":"permata","size":"icon_100x30"},{"name":"Verified by VISA","class":"visa","size":"icon_90x40"},{"name":"MasterCard SecureCode","class":"mastercard","size":"icon_90x40"},{"name":"AMERICAN EXPRESS SafeKey","class":"amex","size":"icon_90x40"}]
```

Comment:
Nice addition but would be more readable if it was better formatted. You can just make a ruby block like so:
```slim
ruby:
  sprite_icons = [
    {},
    {},
    # ...
  ]
```

Reply:
Fixed